crates/ruma-common/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # [unreleased]
 
+Bug fixes:
+
+* Fix `MatrixUri` and `MatrixToUri` parsing
+  * `MatrixToUri` can now be parsed with non-url-encoded room aliases
+  * Both can now be parsed with a query with encoded HTML entities
```

Comment:
Is this something the spec mandates? Encoded HTML entities don't really belong in query strings, right?

Reply:
They don't belong in it but this is something we (and most clients) produce when constructing a message with the markdown parser.

Reply:
In that case, I think we should only have this behind the `compat` feature, if at all. And we should try to fix out own mention extraction somehow.

Reply:
Then I'm gonna leave it out and it'll be part of the mention extraction feature

Reply:
Created #1332.